src/StacBrowser.vue

```diff
@@ -122,6 +122,16 @@ export default {
     ...mapGetters(['displayCatalogTitle']),
     browserVersion() {
       return STAC_BROWSER_VERSION;
+    },
+    appStateAsParams () {
+      const appState = this.$store.state.stateQueryParameters;
```

Comment:
You can move `stateQueryParameters` up to the mapState and then just use `this.stateQueryParameters` in the loop